### ISSUE TITLE
Add a Silhouette coefficient plot for `clusterer` module

### ISSUE THREAD

**Issue by tirthajyoti** [reiinakano/scikit-plot#104]
Hi,

It would be great to have a [Silhouette coefficient](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.silhouette_score.html) plot method to add to the `clusterer` module.
For many situations, it gives a better visual representation of the number of clusters.

Check my article on Medium on this.

**[Clustering metrics better than the elbow-method
](https://towardsdatascience.com/clustering-metrics-better-than-the-elbow-method-6926e1f723a6)**

**Comment by reiinakano** [comment]
This is in the `metrics` module :)

**Comment by tirthajyoti** [comment]
I did not mean the Silhouette analysis plot. I meant a plot where you don't need to know cluster labels. It plots the Silhouette coefficient to determine the optimal number of clusters when you don't know what that is. Much like the elbow plot.
